src/packs/checker/architecture.rs

```diff
@@ -35,40 +35,43 @@ impl Layers {
     }
 }
 
+impl Checker {
+    fn validate_pack(&self, pack: &Pack) -> Option<String> {
+        match &pack.layer {
+            Some(layer) => {
+                if self.layers.layers.contains(layer) {
+                    None
+                } else {
+                    Some(format!(
+                        "Invalid 'layer' option in '{}'. `layer` must be one of the layers defined in `packwerk.yml`",
```

Comment:
So we do have this right, but it was just a `bail` vs. a first-class error right?

(Related to error message: `Could not find one of layer `{}` or layer `{}` in `packwerk.yml`...`)

Reply:
Right. The referenced code above is an architecture layer validation message. The "could not..." is a "check" error.